sdk/iot/Azure.Iot.Hub.Service/tests/TestSettings.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Reflection;
+using Azure.Core.TestFramework;
+using Microsoft.Extensions.Configuration;
+
+namespace Azure.Iot.Hub.Service.Tests
+{
+    /// <summary>
+    /// These are the settings that will be used by the end-to-end tests tests.
+    /// The json files configured in the config will load the settings specific to a user.
+    /// </summary>
+    public class TestSettings
+    {
+        public const string IotHubServiceEnvironmentVariablesPrefix = "IOTHUB";
```

Comment:
just IOT

Reply:
oops, my bad. I'll update this.

Reply:
This would then be: 
`s_iotHubConnectionString = IOT_CONNECTION_STRING";`
That's weird, don't you think?

Reply:
Should this not be changed to "IOT" ?

Reply:
yes, Azad and I spoke offline, and I've updated it to "IOT". You are possibly looking at an outdated commit. :)